source/includes/documentation/_vue_js_elements.md

```diff
@@ -1732,25 +1732,88 @@ For examples of usage see: [JDI vuetify page tests for sparklines](https://githu
 
 ### 5.20 Subheaders
 
-[Vuetify documentation page](https://vuetifyjs.com/en/components/subheaders/)
-
-- __Java__: _com.epam.jdi.light.vuetify.elements.common.Subheader.java_
-
 ```java
-  @Test
-  public void insetTest(){
-      insetSubheader.is().displayed();
-      insetSubheader.is().lightTheme();
-      insetSubheader.is().inset();
-      insetSubheader.is().text("Subheader");
+//@FindBy(css = "#InsetSubheader .v-subheader")
+@UI("#InsetSubheader .v-subheader")
+public static Subheader insetSubheader;
+
+@Test
+public void insetTest() {
```

Comment:
Is it posiible to ask you to refactor all subheader tests = to add descriptions. So that we could have the same approach in all tests. Could you do a favour for us :)

Reply:
Sure, will do it

Reply:
Done